<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GMonteCarloLogNormalPricerWidget</class>
 <widget class="QWidget" name="GMonteCarloLogNormalPricerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GMonteCarloLogNormalPricerWidget</string>
  </property>
  <layout class="QVBoxLayout" name="pVlay">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Payoff</string>
     </property>
     <layout class="QVBoxLayout" name="pPayLay">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="pPayOffStackWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QFormLayout" name="pFormStrike1">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="horizontalSpacing">
           <number>1</number>
          </property>
          <property name="verticalSpacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>1</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QFormLayout" name="pFormStrike2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="horizontalSpacing">
           <number>1</number>
          </property>
          <property name="verticalSpacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>1</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QVBoxLayout" name="pLayoutEquation">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>1</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="pFLay">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>1</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
